storage/container_storage_interface/ephemeral-storage-csi-inline.adoc

```diff
@@ -12,8 +12,6 @@ This feature is only available with supported Container Storage Interface (CSI)
 
 :FeatureName: CSI inline ephemeral volumes
```

Comment:
Can this attribute be removed too? For doc hygiene 😄 I imagine seeing this in the document sometime in the future could be confusing to someone.

Reply:
Good catch! Thanks!